src/test/java/core/basesyntax/service/handlers/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.handlers;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Warehouse;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final Fruit banana = new Fruit("banana");
```

Comment:
```suggestion
    private static final Fruit BANANA = new Fruit("banana");
```
the same for other constants